spec/models/timer_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Timer, type: :model do
+
+  let(:player) { FactoryGirl.create(:player, current_player.id, email: 'pam@rook.com', password: 'BESTKITTY') }
+  let(:game) { FactoryGirl.create(:game, game.id)}
+
+    it "should determine when a player starts" do 
+      expect(DateTime.new).to be_a(DateTime)
+    end
+
+    it "should determine how much time the players have" do
+      # expect(@time_left).to eq(Integer)
```

Comment:
Created the above line - `let(:timer) { FactoryGirl.create(:timer) }` and keep getting this error: 
```
Failures:

  1) Timer should start timer for current players
     Failure/Error: let(:timer) { FactoryGirl.create(:timer)}
     
     ActiveRecord::RecordInvalid:
       Validation failed: Email can't be blank, Password can't be blank
     # ./spec/models/timer_spec.rb:8:in `block (2 levels) in <top (required)>'
     # ./spec/models/timer_spec.rb:20:in `block (2 levels) in <top (required)>'

  2) Timer should stop timer for current players
     Failure/Error: let(:timer) { FactoryGirl.create(:timer)}
     
     ActiveRecord::RecordInvalid:
       Validation failed: Email can't be blank, Password can't be blank
     # ./spec/models/timer_spec.rb:8:in `block (2 levels) in <top (required)>'
     # ./spec/models/timer_spec.rb:25:in `block (2 levels) in <top (required)>'

Finished in 0.83915 seconds (files took 3.77 seconds to load)
34 examples, 2 failures
```
I commented out creating a new timer for each test. I also put email and password in the FactoryGirl.create(:timer, email: 'pam@rook.com', password: 'BIGKITTY' to see if that would work. It didn't.